haproxy_exporter.go

```diff
@@ -352,6 +372,25 @@ loop:
 	return 1
 }
 
+func (e *Exporter) parseInfo(s *bufio.Scanner, ch chan<- prometheus.Metric) {
+	var version, release_date string
+	for s.Scan() {
```

Comment:
You're not checking for errors returned by the scanner. I would make the scanner a function local variable and send an error metric if it fails.

Reply:
Done.

Reply:
I can't see where the scanner errors are checked, can you link to it? Its interface requires the caller to call `Err()` as the loop might exit early in such cases.

Reply:
I'll add an Err() to the return.

Reply:
Ok, added error handling to the scanner.